code/game/machinery/computer/arcade.dm

```diff
@@ -1273,9 +1297,9 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		var/obj/item/bodypart/chopchop = c_user.get_bodypart(which_hand)
 		chopchop.dismember()
 		qdel(chopchop)
+		user?.mind.adjust_experience(/datum/skill/gaming, 100)
```

Comment:
Pay2win mechanics making you a better gamer seems questionable, though if you're smart enough to powergame this then maybe you already fit the meta for an elite gamer.